bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -0,0 +1,37 @@
+{% extends "bare.html" %}
+{% stylesheet 'firefox_new_firstrun' %}
```

Comment:
This should be in a block called `{% block site_css %}{% endblock %}`. Currently this is adding the stylesheets before `<!doctype html>`, which is invalid.